TFlippy_TerritoryControl_Structures_Dev/Entities/Structures/Tavern/Tavern.as

```diff
@@ -102,6 +104,12 @@ void GetButtonsFor(CBlob@ this, CBlob@ caller)
 
 		CButton@ buttonWrite = caller.CreateGenericButton("$icon_paper$", Vec2f(0, -8), this, this.getCommandID("write"), "Rename", params);
 	}
+
+	//toggle tavern light
+	if(player !is null && player.getUsername() == this.get_string("Owner"))
+	{
+		CButton@ buttonLight = caller.CreateGenericButton((this.get_bool("light") ? 27 : 23), Vec2f(12, -3), this, this.getCommandID("sv_togglelight"), "Toggle_Light", params);
```

Comment:
remove buttonLight =

Reply:
what do you mean, all the buttons have an "=" ,is that wrong in the entire thing?